buildSrc/src/main/kotlin/AndroidBuildGradlePlugin.kt

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2024 ForgeRock. All rights reserved.
+ *
+ *  This software may be modified and distributed under the terms
+ *  of the MIT license. See the LICENSE file for details.
+ */
+
+import com.android.build.api.dsl.LibraryExtension
+import org.gradle.api.JavaVersion
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+class AndroidBuildGradlePlugin: Plugin<Project> {
+
+    override fun apply(project: Project) {
+        project.android().apply {
+            compileSdk = 34;
```

Comment:
What do you think about use a constant or property file to avoid the need to change the code directly in a new version?

Reply:
It seems not straightforward to use the variable in toml, may consider it in another story.